packages/remark-shaku-code-annotate/src/index.ts

```diff
@@ -184,11 +199,14 @@ export const remarkShakuCodeAnnotate = (
             shouldHighlighNextSourceLine = false;
             const shouldDim = isDimBlock || shouldDimNextSourceLine;
             shouldDimNextSourceLine = false;
+            const shouldFocus = isFocusBlock || shouldFocusNextSourceLine;
+            shouldFocusNextSourceLine = false;
             const sourceLine = line.line;
-            const prefix = `<div class="line${
-              shouldHighlight ? " highlight" : ""
-            }${shouldDim ? " dim" : ""}">`;
-
+            const focusFlag = hasFocusElement(lines);
+            const highlightClass = shouldHighlight ? " highlight" : "";
+            const focusClass = focusFlag ? (shouldFocus ? "" : " dim") : "";
```

Comment:
let's also add a css class for focus, so that people can easily override the style.